gcloud/pubsub/subscription.py

```diff
@@ -74,9 +74,7 @@ def exists(self):
         """
         conn = self.topic.connection
         try:
-            conn.api_request(method='GET',
-                             path=self.path,
-                             query_params={'fields': 'name'})
```

Comment:
Maybe I mis-attributed another error:  b44aea94188f1b81ba64644afd2d335450a8b6e4 reverts that (and the related test change).